osutil/fshelpers.go

```diff
@@ -0,0 +1,34 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package osutil
+
+import (
+	"syscall"
+)
+
+// FindGidOwning obtains UNIX group ID and name owning file `path`.
+func FindGidOwning(path string) (uint64, error) {
```

Comment:
gid is uint32 in Stat_t, do we have a reason to force uint64?

Reply:
Other calls in `group.go` return uint64. Namely `func FindGid(group string) (uint64, error)` or `func FindUid(username string) (uint64, error)`. I'd like to follow the same pattern, so that whoever is calling this won't have to ponder why this one is different.